### ISSUE TITLE
Mana burst + mana detector + dispenser/dropper

### ISSUE THREAD

**Issue by Janseycraft** [Vazkii/Botania#1777]
The setup is that I send a burst vertically from a pulse mana spreader, through 4 detectors which are adjacent to droppers facing away from the detectors. They dispense the item, but after a unpredictable period of time, certain droppers stop dispensing the items, the burst goes through the detectors (and sends a signal), and once the dropper/ detector has been replaced it will work for normally one burst again and then return to the issue. I have tried changing lenses and dispensers instead of droppers and the issue still occurs. 

Additionally when I have 4 rows of 4 detector + dropper combinations(with force relays and warp lens to move the burst between columns); facing a central empty block column, I see the issue occur normally on the second spreader, sometimes it starts on the first and occasionally it will happen for the column. 

When the issue happens on the second detector, it tends to happen on all 4 of the columns.

I hope that makes sense. Let me know if you need more information.

**Comment by yrsegal** [comment]
A TL;DR since I barely understood this (maybe since i'm tired)

Mana detectors appear to stagnate, appears random. Possible race condition.

**Comment by Janseycraft** [comment]
TL;DR : Mana detector signal will randomly stop interacting with droppers and dispensers, but continues to output a redstone signal to other blocks.

**Comment by yrsegal** [comment]
That works too

**Comment by liftoff** [comment]
I just wanted to add that I too am experiencing this issue.  I setup this *epic* Spectrolus automation but it ends up wasting tons of wool because dispensers/droppers will suddenly stop firing.  Essentially, it seems the mana detectors just plain stop working for certain dispensers/droppers after a while.  Then randomly, sometimes they'll start working again.  It is very strange.

**Comment by williewillus** [comment]
It's because dispensers need to be unpowered and repowered to fire again, so somehow the detector keeps on sending out a signal or fails to cause a block update

**Comment by williewillus** [comment]
some screenshots of the setup might be helpful

**Comment by mindforger** [comment]
@Janseycraft does this especially happen after a server restart? i was having serious blockupdate issues with redstone contraptions, i did yet not find the culprit but it hapopens to several mods redstone emitters not only botania ones

**Comment by yrsegal** [comment]
No, it was a singleplayer world. Let's see if i can pull up the episode...

**Comment by yrsegal** [comment]
https://youtu.be/lIX1EJpHw1M?list=PLewVai9pfh6bbTqxmy3erFsYmPChgHicT

**Comment by yrsegal** [comment]
https://youtu.be/lIX1EJpHw1M?t=21m10s
